src/neptune/new/attributes/namespace.py

```diff
@@ -73,7 +91,9 @@ def to_dict(self) -> Dict[str, Any]:
         return result
 
     def assign(self, value: Union[NamespaceVal, dict, Mapping], wait: bool = False):
-        if not isinstance(value, NamespaceVal):
+        if isinstance(value, argparse.Namespace):
```

Comment:
This is bugfix for assigning `argparse.Namespace` type as `neptune.Namespace` when attribute is already defined.

Due to changes I've made in `Handler.assign` I had to fix it now.

Reply:
In fact I had to change `Handler.assign` implementation again, but bugfix left...